### PR TITLE
adding new rule for events and scans

### PR DESCRIPTION
**Pull request by Slowblitz**
Draft to update the rules to follows the new version of the bep :

- [x] add rules for events
- [x] add rules for scans 
- [x] update test
- [x] update dataset

feel free to add task to the list 
Happy coding

### CODE REVIEW

**Comment by JuliaSprenger** [comment]
ready for review?

**Comment by Slowblitz** [comment]
I think so

**Comment by JuliaSprenger** [comment]
I don't think the docstring.png belongs in this PR. @Slowblitz Did you add this intentionally or by accident?

**Comment by Slowblitz** [comment]
no it's an accident sorry

File: ando/tests/test_AnDOChecker.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ def test_valid_full_metadata(self):
         path = Path(dir_path) / "dataset" / "exp-valid_full_metadata"
         self.assertEqual(CHK.is_valid(path)[0], True)
 
+
+
```
Comment (JuliaSprenger) [inline ando/tests/test_AnDOChecker.py:35]:
```suggestion
```

**Review by JuliaSprenger — APPROVED** [review]

File: ando/tests/test_AnDOChecker.py [hunk 2]
```diff
@@ -31,6 +31,12 @@ def test_valid_full_metadata(self):
         path = Path(dir_path) / "dataset" / "exp-valid_full_metadata"
         self.assertEqual(CHK.is_valid(path)[0], True)
 
+    def test_valid_full_metadata_with_event_scans(self):
+        path = Path(dir_path) / "dataset" / "exp-valid_full_metadata_with_events_scans"
```
[the remark this answers is not in the document]
Comment (Slowblitz) [reply ando/tests/test_AnDOChecker.py]: outdated =)

**Pull request merged**